### PR TITLE
Fixes #15439: Update rudder relay reload command to permit permission check

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/15439

### CODE REVIEW

File: share/commands/relay-reload [hunk 1]
```diff
@@ -7,14 +7,17 @@
 # @man *-q*: run the agent in quiet mode (display only error messages)
 # @man +
 # @man *-c*: run the agent without color output
+# @man +
+# @man *-p*: check permissions of file
```
Comment (amousset) [inline share/commands/relay-reload]: `of files used by relayd`

File: share/commands/relay-reload [hunk 2]
```diff
@@ -26,6 +29,9 @@ while getopts "qciIvd" opt; do
     c)
       clear_colors
       ;;
+    p)
+      chgrp rudder /var/rudder/lib/ssl/allnodescerts.pem
```
Comment (amousset) [inline share/commands/relay-reload]: this should only be done if the file exists (because it won't be there on simple relays)

**Comment by ncharles** [comment]
Commit modified

File: share/commands/relay-reload (hunk 2 above)
Comment (ncharles) [reply share/commands/relay-reload]: note that the reload/start will fail if the file is not there

File: share/commands/relay-reload [hunk 3]
```diff
@@ -26,6 +29,9 @@ while getopts "qciIvd" opt; do
     c)
       clear_colors
       ;;
+    p)
+      if [ -s /var/rudder/lib/ssl/allnodescerts.pem ]; then chgrp rudder /var/rudder/lib/ssl/allnodescerts.pem; fi
```
Comment (amousset) [inline share/commands/relay-reload]: I'd rather use `-f`, this file may be empty.

**Comment by ncharles** [comment]
Commit modified

**Pull request merged**